carts/views.py

```diff
@@ -1,3 +1,36 @@
-from django.shortcuts import render
+import json
+
+from django.views     import View
+from django.http      import JsonResponse
+
+from .models          import Cart
+from core.utils       import signin_decorator
 
 # Create your views here.
+
+class CartView(View):
+    @signin_decorator
+    def post(self, request):
+        try:
+            data       = json.loads(request.body)
+            user       = request.user
+            product_id = data['product_option_id']
+            quantity   = int(data['quantity'])
+
+            if quantity <= 0:
+                return JsonResponse({'message' : 'QUANTITY_ERROR'}, status=400)
+
+            cart, is_created = Cart.objects.get_or_create(
+                user              = user,
+                product_option_id = product_id
+            )
+            cart.quantity = cart.quantity + quantity
```

Comment:
수량이 증가하는 상황만 있는 것 같은데, 만약에 수량을 감소하는 수정이 생기면 어떻게 해야 할 까요?
```suggestion
            cart.quantity = cart.quantity + quantity
```

Reply:
현재 post에 작성한 코드는 상품이 장바구니에 존재 할 경우 수량만 업데이트 시켜주는 로직을 작성해 놓아서
수량 수정은 patch를 이용해서 따로 api를 작성해 줄 거 같습니다!